<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRegExpSearch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Where</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="SmartCheckBox" name="checkBoxUseFileListAsSource">
             <property name="text">
              <string>Use file list as source</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditRootPath"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRootPathBrowse">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>File Search Mask</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEditFileRegExp"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxIsFileRegExpCaseSensitive">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxIsFileRegExpModeEnabled">
           <property name="text">
            <string>RegExp</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxFileIgnoreMask">
        <property name="title">
         <string>File Ignore Mask</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEditFileIgnoreRegExp"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxIsFileIgnoreRegExpCaseSensitive">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxIsFileIgnoreRegExpModeEnabled">
           <property name="text">
            <string>RegExp</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Line Search Mask</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEditLineRegExp"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxIsLineRegExpCaseSensitive">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxIsLineRegExpModeEnabled">
           <property name="text">
            <string>RegExp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="buttonFindFiles">
          <property name="text">
           <string>Find files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonFindLines">
          <property name="text">
           <string>Find lines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonComplexFind">
          <property name="text">
           <string>ComplexSearch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonOpenSelectedFiles">
        <property name="text">
         <string>Open selected files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabFileList">
       <attribute name="title">
        <string>File List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEditFileList">
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabResultFileList">
       <attribute name="title">
        <string>Result file list</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTextEdit" name="textEditResultFileList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFileAndLineList">
       <attribute name="title">
        <string>File and Line List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEditLineList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPresets">
       <attribute name="title">
        <string>Presets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="listWidgetPresets"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPresetSaveAs">
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPresetSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPresetLoad">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPresetRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPreferences">
       <attribute name="title">
        <string>Preferences</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Show ignore masks options</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SmartCheckBox" name="checkBoxIgnoreMaskActive">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>External file viewer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditExternalFileViewer"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrowseExternalFileViewer">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Extra options for external viewer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditExtraOptionExternalFileViewer"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Word wrap</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="SmartCheckBox" name="checkBoxWordWrapEnabled">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_">
          <property name="text">
           <string>Append Lines in Result Window</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="SmartCheckBox" name="checkBoxAppendLinesInResultWindow">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartCheckBox</class>
   <extends>QCheckBox</extends>
   <header>SmartCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
